Which docs are done?

teikjun opened the pull request:
- [x] accessibility.md
- [x] architecture_overview.md
- [x] asynchronous-layout.md
- [x] best-practices.md
- [x] border.md
- [x] cached-values.md
- [x] codegen.md
- [x] common-props.md
- [x] communicating-with-the-ui.md
- [x] community-showcase.md
- [x] component-tree.md
- [x] contributing.md
- [x] custom-layout.md
- [x] debugging.md
- [x] dev-options.md
- [x] diff-sections.md
- [x] dynamic_props.md
- [x] error-boundaries.md
- [x] espresso-testing.md
- [x] handler-testing.md
- [x] events-overview.md
- [x] events-touch-handling.md
- [x] faq.md
- [x] flipper-plugins.md
- [x] getting-started.md
- [x] glossary.md
- [x] group-sections.md
- [x] hscrolls.md
- [x] inc-mount-architecture.md
- [x] inc-mount.md
- [x] injectprop-matching.md
- [x] intro.md
- [x] layout-specs.md
- [x] layout.md
- [x] motivation.md
- [x] mount-specs.md
- [x] onattached-ondetached.md
- [x] prop-matching.md
- [x] props.md
- [x] reconciliation.md
- [x] recycler-collection-component.md
- [x] recycler-component.md
- [x] recycling.md
- [x] repo-structure.md
- [x] rtl.md
- [x] saving-state.md
- [x] sections-architecture.md
- [x] sections-building-blocks.md
- [x] sections-intro.md
- [x] sections-testing.md
- [x] sections-tutorial.md
- [x] sections-view-support.md
- [x] sections-working-ranges.md
- [x] services.md
- [x] state.md
- [x] styles.md
- [x] subcomponent-testing.md
- [x] testing-overview.md
- [x] tests-in-android-studio.md
- [x] tooltips.md
- [x] transition-all-layout.md
- [x] transition-basics.md
- [x] transition-choreography.md
- [x] transition-definitions.md
- [x] transition-faq.md
- [x] transition-key-types.md
- [x] transition-types.md
- [x] tree-props.md
- [x] trigger-events.md
- [x] tutorial.md
- [x] unit-testing.md
- [x] updating-ui.md
- [x] uses.md
- [x] using-components.md
- [x] view-flattening.md
- [x] visibility_handling.md
- [x] widgets.md
- [x] writing-components.md

### minor issues
#### float css is not working:
- event-handler-testing 
- prop-matching
- espresso-testing
#### origin doc has parts that seem like a mistake:
- widget.md
#### consider using tabs for:
- layout-specs
### major issues
#### links to javadoc not working

Drewbi committed:
Docusaurus V2 Init

Drewbi committed:
Add basic Litho content

Drewbi committed:
Merge branch 'master' of https://github.com/facebook/litho into docusaurus-v2-migration

Drewbi committed:
added favicon and default edit location

teikjun committed:
fix some images

Drewbi commented:
# Todo
- [x] Add groovy and kotlin language support
- [x] Add java docs
- [x] Use SCSS instead of CSS to support their SCSS files
- [x] Deploy
- [x] Make landing page responsive
- [x] Make feature text visible by improving contrast
- [x] Fix broken float css in markdown files

Drewbi committed:
Updated dark mode config

Drewbi committed:
Format getting-started

Drewbi committed:
Remove title text

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of https://github.com/MLH-Fellowship/litho into docusaurus-v2-migration

Drewbi committed:
Add groovy and kotlin language support for code blocks

Drewbi committed:
Fix accessibility page table links

Drewbi committed:
Fix intro doc presentation image link

Drewbi committed:
Fix tutorial images and add notes

Drewbi committed:
Change code theme to light mode

teikjun committed:
fix more images and docs

teikjun committed:
fix tree-props

teikjun committed:
fix tutorial

Drewbi committed:
Add info callout to using-components doc

teikjun committed:
fix espresso

Drewbi committed:
Add logo to nav bar

Drewbi committed:
Style home page

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of https://github.com/MLH-Fellowship/litho into docusaurus-v2-migration

teikjun commented:
Todo: 

- [ ] Migrate their SCSS files

teikjun committed:
use scss instead of css

teikjun committed:
use scss modules

teikjun committed:
fix dynamic-props images

teikjun committed:
fix border

teikjun committed:
fix video in architecture-overview

teikjun committed:
resolve merge conflict

teikjun committed:
fix contributing doc

teikjun committed:
fix custom layout

teikjun committed:
fix error boundaries

teikjun committed:
fix event handler testing

teikjun committed:
fix videos and docs

teikjun committed:
fix hscrolls

teikjun committed:
fix all videos in the universe

Commit:
Uses Host#unmount(MountItem) to avoid index mismatch

Summary:
Uses Host#unmount(MountItem) to avoid index mismatch.

Using the index from the layout output is not always correct; for example when disappearing animations move the item to the root the actual mount index would change (i.e. mismatch from the one in its layout output).

To avoid this, it is far safer to call the `Host#unmount(MountItem)` which does not suggest an index; the current implementation finds the current index of the item in Host.

Reviewed By: nicous

Differential Revision: D22159273

fbshipit-source-id: 707405319e3a06912d4a8b0a1e0f1fb92f39a40c

pakoito committed:
Merge logic for TransparencyEnabledCard into Card

Summary: It folds the logic inside TransparencyEnabledCard into Card behind a boolean flag. There are some essential differences in the code that are risky not to reproduce, so I've left the code as it originally was.

Reviewed By: muraziz

Differential Revision: D22093560

fbshipit-source-id: 88dd9c91ef06dbc13d6a3b26f8d14869c63428c4

pakoito committed:
Document the death of TransparencyEnabledCard

Summary: Update the docs to not reference TransparencyEnabledCard anymore.

Reviewed By: muraziz

Differential Revision: D22094012

fbshipit-source-id: 8f6d1e5c9c098343fba738e2f9c4dfa449c8a82c

pakoito committed:
Remove TransparencyEnabledCard from Kotlin API

Summary: We have effectively deprecated `TransparencyEnabledCard`, so let's straight up remove it from the Kotlin API

Reviewed By: muraziz

Differential Revision: D22114231

fbshipit-source-id: 4c718034a08966af9650c8ce71e19c3ce6200ff3

pakoito committed:
Replace TransparencyEnabledCard with Card

Summary: Structural find & replace for all instances. Fixed some docs and references in comments manually.

Reviewed By: muraziz

Differential Revision: D22093559

fbshipit-source-id: 2b72d4cf72607ce3e50c2a20c193a21d3396b904

Drewbi committed:
Update github navbar link

Drewbi committed:
Update navbar hover color

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of https://github.com/MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
resolve merge conflict

Commit:
Removes Host for RenderUnit.Binder APIs

Summary: Removes Host for RenderUnit.Binder APIs

Reviewed By: mihaelao

Differential Revision: D22136791

fbshipit-source-id: 2e0924390f4759f50564d4a35b7192c587e83d24

Commit:
Refactors implementation of Drawable state in ComponentHost

Summary:
Refactors implementation of Drawable state in ComponentHost.

Uses the presence of `LithoMountData` on the `MountItem` to infer if render core is enabled or disable. This way avoids the excess call to set the drawable state when render core is enabled (once in the ComponentHost, and then again in the Binder).

Reviewed By: mihaelao

Differential Revision: D22136718

fbshipit-source-id: 4ff5aac29d4ae3d0d09f9625d7ef1f614789cf9a

teikjun committed:
use admonitions

teikjun committed:
fix broken links

teikjun committed:
fix a mispelling and add more callouts

teikjun committed:
fix broken docs

Drewbi committed:
Change logo to svg and add footer links

Drewbi committed:
Add temporary styling to navbar

Drewbi committed:
Add home page content

Drewbi committed:
Partially style homepage hero and features

Drewbi committed:
Add updated images

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of https://github.com/MLH-Fellowship/litho into docusaurus-v2-migration

Drewbi committed:
Added docsearch

teikjun committed:
fix widget and add infima range of colors

teikjun committed:
make widget buttons nicer

Drewbi committed:
Change button border

Drewbi committed:
Style features

Drewbi committed:
Style button

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of https://github.com/MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
add javadocs

teikjun committed:
fix broken links

teikjun committed:
fix all broken javadoc links

teikjun committed:
resolve merge conflict

teikjun committed:
change route to javadocs

teikjun committed:
match the hero component with the old component's looks

teikjun committed:
fix build errors

teikjun committed:
add border color

teikjun committed:
make banner responsive

Drewbi committed:
Remove unneeded divs

Drewbi committed:
Set base text color

Drewbi committed:
Add sidebar hiding for getting started

Drewbi committed:
Add language tab to layout-spec

Drewbi committed:
Fix mobile nav toggle color

teikjun committed:
fix broken css in docs

teikjun committed:
add border color to buttons

Drewbi committed:
Update prop-matching images

Drewbi committed:
Remove line breaks in prop-matching doc

teikjun committed:
fix border color on deployed site

teikjun committed:
fix fb logo on deployed site

teikjun committed:
change font size of features

Drewbi committed:
Change max width hero text

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of github.com:MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
match litho's landing page background color

teikjun committed:
resize images

Drewbi committed:
Update api links

Drewbi committed:
Add base title for java doc

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of github.com:MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
correct pathname for API

teikjun committed:
make landing page features responsive

teikjun committed:
change some css values

teikjun committed:
organize images into a folder and fix a minor issue

teikjun committed:
modify config

teikjun committed:
add prettier

teikjun committed:
change widget button color

teikjun committed:
add facebook copyright

teikjun committed:
fix video on deployed site

teikjun committed:
try out a fix on deployed site

teikjun committed:
fix broken video links

teikjun committed:
remove remaining jekyll syntax

teikjun committed:
fix broken image links

teikjun committed:
use admonitions

teikjun committed:
use callouts

Drewbi committed:
Update readme and contributing md

Drewbi committed:
Update yarn lock

Drewbi committed:
Update border doc name

Drewbi committed:
Fix common-props quick links

Drewbi committed:
Remove residual target blank

Drewbi committed:
Add documentation contributing link

Drewbi committed:
Fix whitespace and have consistent mdx extensions

Drewbi committed:
Add note callout

Drewbi committed:
Fix home page fonts

teikjun committed:
Revert irrelevant commits

teikjun committed:
Merge branch 'master' of github.com:facebook/litho into docusaurus-v2-migration

teikjun committed:
fix some css issues

teikjun committed:
enable dark mode

Drewbi committed:
Added dark mode code theme

Drewbi committed:
Refactor custom css

teikjun committed:
give images white background for better contrast in dark mode

teikjun committed:
Merge branch 'master' of github.com:MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
Merge branch 'docusaurus-v2-migration' of github.com:MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
use white background for some svgs only

Drewbi committed:
Remove darkmode config fields

teikjun committed:
create component for embedding version in codeblock

Drewbi committed:
Merge branch 'docusaurus-v2-migration' of github.com:MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
use wrapper component for versioned codeblocks

teikjun committed:
change codeblock appearance in dark mode

teikjun committed:
Merge branch 'docusaurus-v2-migration' of github.com:MLH-Fellowship/litho into docusaurus-v2-migration

teikjun committed:
refactor js files

teikjun committed:
use regex for replacement
